test/conv/how-to-run-cnn.md

```diff
@@ -0,0 +1,30 @@
+1. Copy the sample cnnbig.cpp and paste it in puma-compiler/test/
```

Comment:
Shouldn't this (extra steps/caution in CNN) come as a separate section in "how_to_run.md" itself ?
Lines 13-end are repeated.